apps/meteor/app/cloud/server/functions/supportedVersionsToken/supportedVersionsToken.ts

```diff
@@ -134,6 +135,31 @@ const getSupportedVersionsToken = async () => {
 		(response.success && response.result) || undefined,
 	);
 
+	SystemLogger.debug({
+		msg: 'Supported versions',
+		supportedVersionsFromBuild: JSON.stringify(supportedVersionsFromBuild.timestamp),
+		versionsFromLicense: JSON.stringify(versionsFromLicense?.supportedVersions?.timestamp),
+		response: JSON.stringify(response.success && response.result?.timestamp),
```

Comment:
there is no need to call `JSON.stringify` since the logger will already do that before showing the log itself in case the log level is correct